ui/src/app/store/scripts/types.ts

```diff
@@ -0,0 +1,50 @@
+import type { Model } from "app/store/types/model";
+import type { TSFixMe } from "app/base/types";
+
+export type ScriptsHistory = Model & {
+  comment: string | null;
+  created: string;
+  data: string;
+};
+
+export type JSONObject = {
+  // Data from a Django JSONObjectField.
+  [x: string]: TSFixMe;
```

Comment:
This is a JSON object that could contain any structure, not sure how to represent that...